features/movebefore.yml

```diff
@@ -0,0 +1,8 @@
+name: moveBefore()
```

Comment:
Should the ID be "move-before"? I suspect we have a mix of styles now, do our guidelines say one way or the other?

Reply:
We don't have a guideline, but it would be consistent with `structured-clone` at least. Done with 0a377830417471e63c0d039803e794b981258ba2.